### PR TITLE
Release v0.8.0

### LINKED ISSUE #49: Add CHANGELOG.md file
Right now there is no changelog for the project. We should add one as part of the next release.

### PR DESCRIPTION
**Pull request by invidian**
Closes #49

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- Use vendored dependencies while building binaries.
+- `update-agent` example manifests now enforce running agent as root, to ensure agent
+is able to trigger a reboot on the host.
+
+### Fixed
+- Fixed example manifests for Kubernetes 1.17 compatibility.
+- Made operator code compatible with Kubernetes 1.17.
+- Fixed publishing leader acquired event by the operator.
+
+[0.8.0]: https://github.com/flexkube/libflexkube/compare/v0.7.3...v0.8.0
```
[the remark this answers is not in the document]
Comment (invidian) [reply CHANGELOG.md:56]: Oops, nice catch :smile: I was copy pasting from my repo :)

[the remark this answers is not in the document]
Comment (tormath1) [reply CHANGELOG.md:56]: I assumed but I did not find this lines from the `flexkube/libflexkube` changelog so I was wondering. :)

**Review by tormath1 — APPROVED** [review]
LGTM - it would be nice to have the issues / PR link in the generated changelog for the next releases :) Let's ship it :rocket:

**Comment by invidian** [comment]
Fixed couple of typos and grammar mistakes in the changelog, also added mention about rename of Go module.

>it would be nice to have the issues / PR link in the generated changelog for the next releases :)

Thanks for review and suggestion @tormath1 :wink: 

Right now changelog is hand-crafted :hammer_and_wrench: and many items mentioned are spread over several PRs and commits, so it would be quite time consuming to find them all, but indeed, it would be nice to have it automated. I guess "changelog" line on PRs would be good, the same as Kubernetes does.

**Comment by invidian** [comment]
If there is no objections, I'll merge and make RC release. I'd also like to automate pushing of container images before we release.

**Comment by tormath1** [comment]
@invidian let's go for merging this. Images pushing can be done in a follow-up PR.

> I guess "changelog" line on PRs would be good, the same as Kubernetes does.

This is also how we do on the mantle side, we recently added a changelog too. It's currently done manually because it's not that "hard" to maintain, you just need to think about adding your entry when you submit a PR. (https://github.com/flatcar-linux/mantle/blob/flatcar-master/CHANGELOG.md#unreleased)

**Review by tormath1 — APPROVED** [review]

**Pull request merged**